narwhals/series.py

```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this Series are in the other Series.
```

Comment:
```
Check if elements of this Series are in a given sequence.
```

we don't support `other` being a Series of lists here as Polars does, we can only accept a single sequence

Reply:
thanks, I updated summary.